bucket/ktlint.json

```diff
@@ -0,0 +1,19 @@
+{
+    "version": "1.0.0",
+    "description": "An anti-bikeshedding Kotlin linter with built-in formatter",
+    "homepage": "https://github.com/pinterest/ktlint/",
+    "license": "MIT",
+    "suggest": {
+        "JDK": [
+            "java/oraclejdk",
+            "java/openjdk"
+        ]
+    },
+    "url": "https://github.com/pinterest/ktlint/releases/download/1.0.0/ktlint#/ktlint.jar",
+    "hash": "5d19ef51d198eecf1280fb167192ed1e1374c70eb9311abceb89207ad06481dd",
+    "bin": "ktlint.jar",
+    "checkver": "github",
+    "autoupdate": {
+        "url": "https://github.com/pinterest/ktlint/releases/download/$version/ktlint#/ktlint.jar"
```

Comment:
```suggestion
        "url": "https://github.com/pinterest/ktlint/releases/download/$version/ktlint#/ktlint.jar",
        "hash": {
            "url": "$url.md5"
        }
```

Reply:
Thanks for your review. However,
1. [ktlint@0.48.2](https://github.com/pinterest/ktlint/releases/tag/0.48.2) seems to be the last release utilizing `md5` hashes, but not anymore.
![image](https://github.com/ScoopInstaller/Main/assets/11015675/e5d5363a-2a6b-45b4-b8ba-6df4d2272e6c)

2. They only provide a PGP signature (`ktlint.asc` contains PGP signature) mentioned in their [documentation](https://pinterest.github.io/ktlint/1.0.0/install/cli/). 

Would you still like to add this section or other modifications?
```        
"hash": {
            "url": "$url.md5"
 }
```